mlrun/model_monitoring/helpers.py

```diff
@@ -45,18 +45,21 @@ class _BatchDict(typing.TypedDict):
 
 
 def get_stream_path(
-    project: str, function_name: str = mm_constants.MonitoringFunctionNames.STREAM
+    project: str,
+    function_name: str = mm_constants.MonitoringFunctionNames.STREAM,
+    stream_uri: typing.Optional[str] = None,
```

Comment:
I didn't see any use of that param except for the tests. It is only for the test?

Reply:
Yep it is only for the test..